mmedit/models/mattors/base_mattor.py

```diff
@@ -129,7 +129,6 @@ def restore_shape(self, pred_alpha, meta):
         """
         ori_trimap = meta[0]['ori_trimap'].squeeze()
         ori_h, ori_w = meta[0]['merged_ori_shape'][:2]
-
```

Comment:
Is this change necessary?

Reply:
> Is this change necessary?

Sorry, it's my fault, I have retrieved this change.